x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce.json

```diff
@@ -92,6 +92,10 @@
       [
         "order_date",
         "desc"
+      ],
+      [
+        "order_id",
+        "desc"
```

Comment:
@wayneseymour with this change the sort order is now stable, however, it's different than before, so snapshots had to be re-generated